src/orquestra/quantum/operators/_openfermion_utils/config.py

```diff
@@ -1,3 +1,6 @@
+################################################################################
+# © Copyright 2022 Zapata Computing Inc.
```

Comment:
This file already has a copyright notice

Reply:
Fixed in https://github.com/zapatacomputing/orquestra-quantum/pull/53/commits/98a53a9b5102d794e3b8189caaa4e6db7f364a38